templates/ldap/openldap-svc.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.georchestra.webapps.openldap.enabled -}}
+{{- $webapp := .Values.georchestra.webapps.openldap -}}
```

Comment:
is it worth it here ? It's not even used after this initial test

Remark applies for other files too

Reply:
well it's for future-proof. for the future in case we need to use the variable it's there, it's available.